src/basics.md

```diff
@@ -122,11 +124,30 @@ extern crate rand;
 use rand::Rng;
 
 fn main() {
+    // Each thread has an automatically-initialised random number generator:
     let mut rng = rand::thread_rng();
-    println!("Random f64: {}", rng.gen::<f64>());
+    
+    // Integers are uniformly distributed over the type's whole range:
+    let n1: u32 = rng.gen();
+    let n2: u16 = rng.gen();
+    let n3: u8 = rng.gen();
+    println!("Random u32, u16, u8: {}, {}, {}", n1, n2, n3);
+    println!("Random i32: {}", rng.gen::<i32>());
+    
+    // Floating point numbers are uniformly distributed in the half-open range [0, 1)
+    println!("Random float: {}", rng.gen::<f64>());
+    println!("Random floats: {:?}", (0..5).map(|_| rng.gen()).collect::<Vec<f32>>());
 }
 ```
 
+Example output:
```

Comment:
>  Is this planned?

Sorry I wanted to mention it as the main reason but it slipped my mind. Yep the run button is already implemented but disabled due to rust playpen historically not supporting crates. Currently we are coordinating list of supported crates with playpen guys https://github.com/brson/rust-cookbook/issues/87